kolibri/plugins/coach/assets/src/views/plan/LessonResourceSelectionPage/LessonContentCard/index.vue

```diff
@@ -21,49 +14,49 @@
         :style="{ color: $themeTokens.text }"
       >
         <h3
-          v-if="!windowIsSmall"
           class="title"
           dir="auto"
         >
-          <KLabeledIcon :label="title">
-            <template #icon>
-              <ContentIcon :kind="kind" />
-            </template>
-          </KLabeledIcon>
+          <KTextTruncator
+            :text="title"
+            :maxLines="2"
+          />
         </h3>
-        <h3
-          v-if="windowIsSmall"
-          dir="auto"
-        >
-          <KLabeledIcon :label="title">
-            <template #icon>
-              <ContentIcon :kind="kind" />
-            </template>
-          </KLabeledIcon>
-        </h3>
-        <div
-          v-if="message"
-          class="message"
-          :style="{ color: $themeTokens.text }"
-        >
-          {{ message }}
-        </div>
       </div>
       <KTextTruncator
         v-if="!windowIsSmall"
         :text="description"
         :maxLines="3"
         class="description"
+        :style="{ color: $themeTokens.annotation }"
       />
       <div>
+        <span
+          v-if="message"
+          class="message"
+          :style="{ color: $themeTokens.annotation }"
+        >
+          {{ message }}
+        </span>
         <CoachContentLabel
           class="coach-content-label"
           :value="numCoachContents"
           :isTopic="isTopic"
         />
       </div>
       <slot name="notice"></slot>
+      <LearningActivityChip
+        v-if="isLeaf"
+        :kind="kind"
+        class="chip"
+      />
     </div>
+    <CardThumbnail
+      v-if="!windowIsSmall"
+      class="thumbnail"
+      :thumbnail="thumbnail"
+      :kind="kind"
```

Comment:
It seems that no updates have been made to the CardThumbnail component, but we are now passing a learning activity array to this, rather than a content kind, I haven't checked, but I expect this to end poorly.